cli/docker/docker.go

```diff
@@ -133,13 +135,20 @@ func (handler *Handler) GetImageFlavor(c config.JobConfiguration) string {
 	if c.Image.Version != "" {
 		tag = c.Image.Version
 	}
-	return fmt.Sprintf("%s:%s", c.Image.Base, tag)
+	defaultRegistry := "docker.io"
+	imageName := fmt.Sprintf("%s:%s", c.Image.Base, tag)
+        match, _ := regexp.MatchString(`.+\/.*\/.*`, imageName)
```

Comment:
Hm, I guess this relies on there being an org name as part of the image name?

Officially "blessed" images (e.g. https://hub.docker.com/_/ubuntu) are at the root and don't have an org name.
So if we use that as an example and the user had a private repository simply as a security gateway or cache, this wouldn't work: `my.private.registry/my-image:latest`

Reply:
@alexplischke yes, I was investigating those case scenarios. I'm going to be updating regex and adding unit tests soon.